packages/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -17,6 +17,8 @@ interface IInMeetingActions {
   canUnsetMuteOnEntry?: boolean;
   canSetDisallowUnmute?: boolean;
   canUnsetDisallowUnmute?: boolean;
+  canSetMuted?: boolean;
+  canUnsetMuted?: boolean;
```

Comment:
question, do you think these would be more clear as canMuteAll/canUnmuteAll?

Reply:
For some reason, the locus controls endpoint accepts `muted: true` as the way to mute all participants. Seems like they could have chosen something better. Anyway, some of the logic we use to determine what data to send to /controls depends on the keys we send them in the payload. In this case, "muted". Long story short, I was just sticking with the current naming conventions.